src/main/java/com/salesforce/dynamodbv2/mt/mappers/metadata/DynamoTableDescriptionImpl.java

```diff
@@ -148,18 +148,13 @@ public String getTableName() {
         return new ArrayList<>(lsiMap.values());
     }
 
-    @Override
-    public Optional<DynamoSecondaryIndex> getLsi(String indexName) {
-        return Optional.ofNullable(lsiMap.get(indexName));
-    }
-
     @Override
     public DynamoSecondaryIndex findSi(String indexName) {
         Optional<DynamoSecondaryIndex> si = getGsi(indexName);
-        if (!si.isPresent()) {
-            si = getLsi(indexName);
```

Comment:
Unused declaration ... basically made the `getLsi()` method private, but then it was only used once, so I inlined it.

Reply:
I prefer the parallelism offered by a separate `getLsi` method (vis-à-vis `getGsi`)—private or otherwise. (that's certainly just an opinion, of course!)

Reply:
fixed in ce9edfb34bb6fb9b893c494b968927f2974c8aed